mobile/Features/Other/Components/FilterAndBannedTags.jsx

```diff
@@ -0,0 +1,64 @@
+import {
+  View, Text, TouchableOpacity, Button,
+} from 'react-native';
+import { useState } from 'react';
+import FilterAndBannedTagsStyle from './FilterAndBannedTagsStyle';
+
+export default function FilterAndBannedTags({ navigation, screenOption }) {
+  const tags = ['abuse', 'addiction', 'anger',
+    'anger management', 'anxiety',
+    'bipolar disorder', 'body dysmorphia',
+    'body image', 'breakups', 'burnout'];
+
+  const [selectedTags, setSelectedTags] = useState([]);
+
+  const toggleTag = (tag) => {
+    if (selectedTags.includes(tag)) {
+      setSelectedTags(selectedTags.filter((selected) => selected !== tag));
+    } else {
+      setSelectedTags([...selectedTags, tag]);
+    }
+  };
+
+  const clearAll = () => {
+    setSelectedTags([]);
+  };
+
+  // const applyAll = () => {
+  //   setSelectedTags([...tags]);
+  // };
```

Comment:
delete! i assume its not useful anymore; and when someone else picks up the backend integration i think they'll be able to figure it out themselves